pkg/components/renderer/renderer.go

```diff
@@ -114,7 +91,74 @@ func RenderToPng(params *RenderOpts) (string, error) {
 		cmdArgs = append([]string{fmt.Sprintf("--output-encoding=%s", params.Encoding)}, cmdArgs...)
 	}
 
-	cmd := exec.Command(binPath, cmdArgs...)
+	rendererLog.Debug(`rendering using phantomjs:`, binPath, cmdArgs)
+
+	return exec.Command(binPath, cmdArgs...)
+}
+
+func renderWithChrome(url, pngPath, domain, renderKey string, timeout int, params *RenderOpts) *exec.Cmd {
+	cmdArgs := []string{
+		setting.RenderingHeadlessScript,
+		"url=" + url,
+		"width=" + params.Width,
+		"height=" + params.Height,
+		"png=" + pngPath,
+		"domain=" + domain,
+		"timeout=" + strconv.Itoa(timeout),
+		"renderKey=" + renderKey,
+	}
+
+	if !setting.RenderingHeadless {
+		cmdArgs = append(cmdArgs, "headless=false")
+	}
+
+	if setting.RenderingSlowMo > 0 {
+		cmdArgs = append(cmdArgs, fmt.Sprintf("slowmo=%d", setting.RenderingSlowMo))
+	}
+
+	// hehe hardcode node for now
+	return exec.Command("node", cmdArgs...)
```

Comment:
Does this means Grafana will require nodejs to start headless chrome? We cannot have nodejs as an requirement for running Grafana.

Reply:
I am a noob, but how do I execute my `screenshot.js` file if not with node?

Reply:
Maybe there is a way to control headless chrome from go? like https://github.com/chromedp/chromedp